lib/github_api/client/git_data/tags.rb

```diff
@@ -4,6 +4,7 @@ module Github
   class Client::GitData::Tags < API
     # This tags api only deals with tag objects -
     # so only annotated tags, not lightweight tags.
+    # Refer https://developer.github.com/v3/git/tags/#parameters
 
     VALID_TAG_PARAM_NAMES = %w[
```

Comment:
I would go as far as removing this all together. Validating options is devil's work and creates tons of headaches going forward. Anytime endpoints params change things need to be updated in the client. It's better to delete the validation.